### DISCUSSION TITLE
Claim organization name<title>

### DISCUSSION THREAD

**Discussion by bothorsen** [community/community#119761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I just tried to create a company account with my company name (Viking Software), but it was taken.

The organization has not had any activity since 2016 where a single thing was done (probably create a repo). Nothing else is there.

I may even have made this myself at the time, since my company already had the name back then. But I tried logging in with my old email address at the time and did not find an account, and my current account does not have access to this organization.

How can I take over the organization name? As far as I know, we are the only company in the world with this name now.

**Comment by ebndev** [comment]
Hi @bothorsen,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
